packages/rspack/src/fileSystem.ts

```diff
@@ -1,66 +1,40 @@
 import util from "util";
-import { join } from "path";
+import { ThreadsafeNodeFS } from "@rspack/binding";
+
 import { memoizeFn } from "./util/memoize";
+import { OutputFileSystem, rmrf, mkdirp } from "./util/fs";
 
-export interface ThreadsafeWritableNodeFS {
-	writeFile: (...args: any[]) => any;
-	removeFile: (...args: any[]) => any;
-	mkdir: (...args: any[]) => any;
-	mkdirp: (...args: any[]) => any;
-	removeDirAll: (...args: any[]) => any;
-}
+const NOOP_FILESYSTEM: ThreadsafeNodeFS = {
+	writeFile() {},
```

Comment:
Seems arugments lost

Reply:
`NOOP_FILESYSTEM` is for the child compiler where it's assets are emitted using parent compilation.